www/include/configuration/configGenerate/xml/postcommand.php

```diff
@@ -71,23 +73,19 @@
             continue;
         }
         $str .= "<br/><strong>{$row['name']}</strong><br/>";
+
         foreach ($commands as $command) {
-            $output = array();
-            exec($command['command_line'], $output, $result);
-            $resultColor = "green";
-            if ($result != 0) {
-                $resultColor = "red";
-                $ok = false;
+            if ($fh = @fopen($centcorePipe, 'a+')) {
```

Comment:
not sure, what happen when you use fopen / fwrite on a file if you don't have the right to

Reply:
as you told me, this should be fine ;)